src/eckit/parser/YAMLParser.cc

```diff
@@ -425,12 +426,47 @@ Value YAMLParser::parseNumber() {
 }
 
 
+std::string ignoreChar(const std::string& s, const char ignore) {
+    std::string str = s;
+    str.erase(remove(str.begin(), str.end(), ignore), str.end());
+    return str;
+}
+
+
+long long parseSexagesimal(const std::string& s) {
```

Comment:
Do we want to support Sexagesimal literals? They have been removed in YAML 1.2 IIRC, and cause really awkward bugs if you ever want to have (say) MAC addresses in a YAML... (i.e. they can misparse ONLY if they contain no hex digits A-F)

Reply:
I have removed this as commented below.